e2e/proxy-pallet/proxy.test.ts

```diff
@@ -93,7 +93,6 @@ describe('Proxy', function () {
       // Filtered Out ExtrinsicFailed
       await assert.rejects(extrinsic.signAndSend(), {
         name: 'NotProxy',
-        message: /Sender is not a proxy of the account to be proxied./,
```

Comment:
These messages are not sent back when running against Testnet on Paseo

Reply:
It is the way the builds work. It has to do with how much is compiled into the runtime. I don't recall the specific details.